api_spec.yaml

```diff
@@ -199,25 +203,42 @@ paths:
                         $ref: '#/components/schemas/statusOK'
                       clientId:
```

Comment:
no need for separate status codes for expiry and invalid

Reply:
changes made